### PR TITLE
[Design] Add SQL Server Cache to the Session Sample

### PR DESCRIPTION
**Pull request by kichalla**
@Tratcher

### CODE REVIEW

File: samples/SessionSample/Startup.cs [hunk 1]
```diff
@@ -4,6 +4,7 @@
 using System;
 using Microsoft.AspNet.Builder;
 using Microsoft.AspNet.Http;
+using Microsoft.Framework.Caching.Distributed;
```
Comment (Tratcher) [inline samples/SessionSample/Startup.cs]: This isn't the right namespace is it?

**Comment by Tratcher** [comment]
This looks fine, pending my comments on the other PR.

**Comment by hishamco** [comment]
@kichalla while you are mentioned SQL Server cache, any comment regarding this issue #21

**Comment by kichalla** [comment]
commit 28dfa0f0c42a926b9666d29d3bb2110b9ff30738

**Pull request closed without merging**